scripts/genproto.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Generate rkt protobuf bindings.
+# Run from repository root.
+#
+set -e
+
+if ! [[ "$0" =~ "scripts/genproto.sh" ]]; then
+	echo "must be run from repository root"
+	exit 255
+fi
+
+if ! [[ $(protoc --version) =~ "3.0.0" ]]; then
+	echo "could not find protoc 3.0.0, is it installed + in PATH?"
+	exit 255
+fi
+
+export GOPATH=$(mktemp -d)
+export PATH=${GOPATH}/bin:${PATH}
+
+trap 'rm -rf "${GOPATH}"' EXIT
+
+# git (sha) version of golang/protobuf
+GO_PROTOBUF_SHA="2402d76f3d41f928c7902a765dfc872356dd3aad"
+
+echo "installing golang/protobuf using GOPATH=${GOPATH}"
+go get -u github.com/golang/protobuf/{proto,protoc-gen-go}
```

Comment:
You copied my script and removed my [TODO](https://github.com/coreos/etcd/blob/1e61243fd73437ad2bd0d29bf08fcbe998c48d63/scripts/genproto.sh#L43) without doing it? :P